### PR TITLE
Create Income and Create Expense mutations

### PR DESCRIPTION
**Pull request by ben-rosner-williamsburg**
### Keypoints:
- Add createIncome mutation
- Add createExpense mutation
- Implement both mutations in code.
- Resolve merge conflict

### Related Issue(s)
closes df-69
closes df-70

### Checklist:
- [ ] New/Modified Tests are Passing
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings or errors.

#### Additional Notes:
Error is coming up for Apollo
https://www.apollographql.com/docs/react/errors/#%7B%22version%22%3A%223.9.2%22%2C%22message%22%3A14%2C%22args%22%3A%5B%22user%22%2C%22Query%22%2C%22either%20ensure%20all%20objects%20of%20type%20User%20have%20an%20ID%20or%20a%20custom%20merge%20function%2C%20or%20%22%2C%22Query.user%22%2C%22%7B%5Cn%20%20%5C%22__typename%5C%22%3A%20%5C%22User%5C%22%2C%5Cn%20%20%5C%22currentExpenses%5C%22%3A%20%7B%5Cn%20%20%20%20%5C%22__typename%5C%22%3A%20%5C%22CurrentExpenses%5C%22%2C%5Cn%20%20%20%20%5C%22amount%5C%22%3A%2011434%2C%5Cn%20%20%20%20%5C%22pctChange%5C%22%3A%20-17.912269366070788%5Cn%20%20%7D%5Cn%7D%22%2C%22%7B%5Cn%20%20%5C%22__ref%5C%22%3A%20%5C%22User%3A2%5C%22%5Cn%7D%22%5D%7D

### CODE REVIEW

**Comment by ben-rosner-williamsburg** [comment]
@MaryBruff

**Comment by memcmahon** [comment]
Code looks good - though I find it interesting that you ran into a merge conflict in your app.js file - seems like git should have handled that fine 😆 

Generally, Its interesting to see only 1 of 4 tasks completed on a PR that is under review-request.  I think you might want to add some comments as to the status of things like testing, documentation, and code comments.  Since its included in your PR template, its worth a mention.  This also could be an opportunity to revisit what is expected in a PR template!

**Comment by ben-rosner-williamsburg** [comment]
@

> Code looks good - though I find it interesting that you ran into a merge conflict in your app.js file - seems like git should have handled that fine 😆
> 
> Generally, Its interesting to see only 1 of 4 tasks completed on a PR that is under review-request. I think you might want to add some comments as to the status of things like testing, documentation, and code comments. Since its included in your PR template, its worth a mention. This also could be an opportunity to revisit what is expected in a PR template!

That's good feedback! Thank you. I will bring this up at standup.

File: src/fetchCalls.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { GET_EXPENSES } from "./queries/getExpenses";
 import { GET_INCOMES } from "./queries/getIncomes";
 import { GET_TRANSACTIONS } from "./queries/getTransactions";
 import { GET_USER_CASH_FLOW } from "./queries/getUserCashFlow";
+import { CREATE_EXPENSE } from "./components/apollo-client/mutations/createExpense";
```
Comment (MaryBruff) [inline src/fetchCalls.js:7]: With the current App set up, this fetch file is no longer in use so no need to import anything new in here. I didn't want to delete the file but maybe we can in a future branch

Comment (ben-rosner-williamsburg) [reply src/fetchCalls.js:7]: @MaryBruff Yep that's a good point. Thank you!

**Comment by ben-rosner-williamsburg** [comment]
@MaryBruff @chisPmama This has been updated. Please review latest push.

**Pull request closed without merging**